tests/unit/records/test_records_receivers.py

```diff
@@ -354,7 +354,46 @@ def test_assign_phonetic_block_handles_jimmy():
     assert expected == result
 
 
-def test_assign_phonetick_block_ignores_malformed_names():
+def test_assign_phonetic_block_handles_two_authors_with_the_same_name():
```

Comment:
maybe also throw in some author with an invalid name?

Reply:
Is [this guy](https://github.com/inspirehep/inspire-next/blob/a87ab4601c2a239affa056500f9c9954ca547d90/tests/unit/records/test_records_receivers.py#L357-L377) enough, or do you want more?